components/ViewBillsOnHomePage/ViewBillsOnHomePage.js

```diff
@@ -23,8 +23,13 @@ const BillRow = props => {
             <Button variant="primary">{formatBillId(bill.BillNumber)}</Button>
           </Wrap>
         </td>
-        <td>{bill.Title}</td>
-        <td>{fullBill.nextHearingAt?.toDate().toLocaleDateString()}</td>
+        <td>{bill.Title.substring(0, 40)}...</td>
+        <td>
+          {fullBill.nextHearingAt
+            ?.toDate()
```

Comment:
this is already a date string so I think we can revert this line

Reply:
done